couchbase/example/src/main/java/example/springdata/couchbase/model/Airline.java

```diff
@@ -0,0 +1,108 @@
+package example.springdata.couchbase.model;
+
+import com.couchbase.client.java.repository.annotation.Field;
+import com.couchbase.client.java.repository.annotation.Id;
+import org.springframework.data.couchbase.core.mapping.Document;
+
+@Document
+public class Airline {
```

Comment:
Please use Lombok annotation to avoid all the boilerplate for getters, setters, equals, hashCode.